WebContent/index.html

```diff
@@ -16,6 +16,33 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=0, maximum-scale=1, minimum-scale=1">
     <title>JES Explorer</title>
 
+    <script>
+      if(exports){
+        var ZoweZLUX_tempExports = exports;
+      }
+      var exports = {"__esModule": true};
+    </script>
+    <script type="text/javascript" src="../../../../../lib/org.zowe.zlux.logger/0.9.0/logger.js"></script>
+    <script type="text/javascript" src="../../../plugins/org.zowe.zlux.bootstrap/web/iframe-adapter.js"></script>
+    <script>
+      function handleMessageAdded(message){
+      }
+      function handleMessageRemoved(id) {
+      }
+
+      async function sendJesNotifications(message){
```

Comment:
Does this need to be at the scope of the window? can't it reside in WebContent/js/components/Snackbar.js ?

Reply:
I hate to be picky but I feel like this function should also be more accurately named that we're sending the notification to the desktop.

Reply:
right now i am registering them at windows scope, i think we need one object at highlevel, and can register all functions there, haven't tried this though, right now we register all handler and call back at windows level
ZoweZLUX.notificationManager.addMessageHandler(window) =>
ZoweZLUX.notificationManager.addMessageHandler(zluxHookObject)

https://github.com/zowe/explorer-jes/blob/1e41610c602f077f4664b2be37738fb08958e565/WebContent/zlux-hooks/register-handler.js#L24

Reply:
> I hate to be picky but I feel like this function should also be more accurately named that we're sending the notification to the desktop.

modified name